apps/bfd-model/bfd-model-rif/src/main/java/gov/cms/bfd/model/rif/parse/RifParsingUtils.java

```diff
@@ -56,7 +61,11 @@ public static CSVParser createCsvParser(CSVFormat csvFormat, RifFile file) {
    */
   public static CSVParser createCsvParser(
       CSVFormat csvFormat, InputStream fileStream, Charset charset) {
-    BOMInputStream fileStreamWithoutBom = new BOMInputStream(fileStream, false);
+
+    InputStream fileStreamStrippedOfBackslashes = new ReplacingInputStream(fileStream, "\\|", "|");
```

Comment:
Should we consider moving this after the BOMInputStream and before the final InputStream

Reply:
done